policies/pol/sample_openconfig.pol

```diff
@@ -24,10 +24,8 @@ term accept-from-googledns {
 term sport-dport {
   comment:: "DENY SMTP."
   destination-address:: INTERNAL
-  source-address:: BOGON
```

Comment:
Why change this one?

Reply:
Oops, my bad, that was a quick test I didn't mean to commit. Fixed.